README.md

```diff
@@ -240,3 +243,21 @@ describe('ComponentToTest', () => {
   }
 }
 ```
+
+
+## Release cycle:
+
+### Pre-releases
+
+To create a pre-release one can push/merge changes to branch next. This triggers actions to automatically create a pre-release.
+you can use the current pre-release by using `@next` as version in your project package.json.
+
+### Releases
+
+1. Merge your branch/changes into main-branch
+2. bump version in package.json with `npm version [<newversion> | major | minor | patch | premajor | preminor | prepatch | prerelease [--preid=<prerelease-id>]`
```

Comment:
* Start of list items should be all uppercase
* I would not include all esoteric arguments here, just `npm version [<newversion> | major | minor | patch`

Reply:
done